### DISCUSSION TITLE
How to Upload the image of my academic status? (not using webcam or smartphone)

### DISCUSSION THREAD

**Discussion by jihoonwang98** [community/community#127953] [GitHub Education]
### Hi everyone,

<img width="884" alt="image" src="https://github.com/community/community/assets/51291420/e5828089-43cd-4a99-85c2-3400e0273e1e">

The image quality from webcams or smartphones does not clearly show the date. Is there a way to upload the image?

Why isn't there an option to upload images?

**Comment by WzK000** [comment]
I have the same issue. I'm getting rejected due my attachment quality

**Comment by samus-aran** [comment]
Hi @jihoonwang98 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I can only use my Front Camera to take a photo.](https://github.com/orgs/community/discussions/111352#use-front-camera)

> We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a phone to capture.

1. Please ensure that you have allowed camera access on the browser you're using to apply. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).
2. If you can, print off your evidence and take a photo of this copy.
3. Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone.
4. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
5. Alternatively you may use another mobile device or computer.

**Comment by nateybear** [comment]
Hi @samus-aran,

Is there no way to upload a document? My school provides electronic proof of enrollment, but I don't have any way to print it out and then take a picture of it.

**Comment by sleepingcat4** [comment]
@samus-aran I want to redeem my Github student pack unfortunately I'm not able to since I can't access my back camera on phone. It doesn't make sense why GitHub asks for using front camera on phone. LIKE IT's CLEARLY STUPID. Do you have a fix for this?
